app/assets/stylesheets/style.scss

```diff
@@ -62,7 +62,17 @@ It was originally created by Caroline Hadilaksono: http://www.hadilaksono.com
 .map-list div:nth-child(4n+1){
   clear: left;
 }
-
+.e2_48 { 
```

Comment:
Hi, could we use a better class name for this? not important but better for people looking at the file to figure out where its used

Reply:
Yeah! That will be great 😃

Reply:
@cesswairimu  I had made the required changes. Kindly review it! Thank You!